parm/metplus_config/plots/cam/grid2obs/cam_firewxnest_plots_py_plotting.config_ctc_pbl

```diff
@@ -77,7 +77,7 @@ set -x
 # Case type settings are used to check if settings are allowed. **
 export VERIF_CASE="grid2obs"
 export VERIF_TYPE="conus_sfc"
-export MODELS="nam_firewxnest"
+export MODELS="nam_firewxnest, rrfs_firewxnest"
```

Comment:
Similar question as above, just wondering the context here and the differences between the model strings used at different parts of the EVS workflow

Reply:
I'll add here re: different model strings used at different parts of the EVS workflow:  I'm not sure that "MODELNAME" is an NCO standard, but I believe it's an EVS standard; each job has a MODELNAME defined in the driver and ecf script.  For plots jobs, this is often set to something like "cam" because multiple models are being plotted, whereas for stats this will usually be set to a single model (e.g., "hrrr").  Elsewhere in the code, other model lists like the one linked here exist to specify things like, what to plot, or what stats to look for before plotting.